specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/examples/Output_Create_PowerBI.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "api-version": "2017-04-01-preview",
+    "subscriptionId": "56b5e0a9-b645-407d-99b0-c64f86013e3d",
+    "resourceGroupName": "sjrg",
+    "jobName": "sj2331",
+    "outputName": "output3022",
+    "output": {
+      "properties": {
+        "datasource": {
+          "type": "PowerBI",
+          "properties": {
+            "dataset": "someDataset",
+            "table": "someTable",
+            "refreshToken": "someRefreshToken==",
+            "tokenUserPrincipalName": "bobsmith@contoso.com",
+            "tokenUserDisplayName": "Bob Smith",
+            "groupId": "ac40305e-3e8d-43ac-8161-c33799f43e95",
+            "groupName": "MyPowerBIGroup",
+            "authenticationMode": "ConnectionString"
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {
+        "ETag": "ccf8b864-259e-43c5-a628-ba10858b2c07"
+      },
+      "body": {
+        "id": "/subscriptions/56b5e0a9-b645-407d-99b0-c64f86013e3d/resourceGroups/sjrg/providers/Microsoft.StreamAnalytics/streamingjobs/sj2331/outputs/output3022",
+        "name": "output3022",
+        "type": "Microsoft.StreamAnalytics/streamingjobs/outputs",
+        "properties": {
+          "datasource": {
+            "type": "PowerBI",
+            "properties": {
+              "dataset": "someDataset",
+              "table": "someTable",
+              "tokenUserPrincipalName": "bobsmith@contoso.com",
+              "tokenUserDisplayName": "Bob Smith",
+              "groupId": "ac40305e-3e8d-43ac-8161-c33799f43e95",
+              "groupName": "MyPowerBIGroup",
+              "authenticationMode": "ConnectionString"
```

Comment:
You should make sure examples have reasonable values. ConnectionString doesn't make sense for a PowerBI output